### DISCUSSION TITLE
My Color Scheme

### DISCUSSION THREAD

**Discussion by JEBurroughs** [mriscoc/Ender3V2S1#693] [General]
I used the Giadej compilation  firmware for the display. I have no clue how to make it a gcode deal so here is what ya get. I love it. Very eye friendly.
![5](https://user-images.githubusercontent.com/79555834/215283844-538fac8f-feaa-4f56-9c45-3535078e91be.jpg)
![1](https://user-images.githubusercontent.com/79555834/215279815-521dba1d-cb27-4bd6-a857-c95e8f5ec6de.jpg)
![4](https://user-images.githubusercontent.com/79555834/215279819-f651f711-3129-41ee-a5cd-42ff28a00059.jpg)
![2](https://user-images.githubusercontent.com/79555834/215279821-1b2f17e4-b284-4759-8e58-c3cd0ecd316f.png)

**Comment by mriscoc** [comment]
Nice
